docker-entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cd /project
+
+npm run build
```

Comment:
You should be running `npm run dev` for development, not `npm run build`. `build` is for build the JS/CSS files, where `dev` runs Webpack development server and takes care of hot reload of the assets.